tls/s2n_cipher_preferences.c

```diff
@@ -181,6 +181,35 @@ const struct s2n_cipher_preferences cipher_preferences_20170210 = {
     .minimum_protocol_version = S2N_TLS10
 };
 
+/* Preferences optimized for interop. DHE and 3DES are added(at the lowest preference). */
```

Comment:
Is there a reason why ChaCha isn't in this list? From reading the comment I was thinking "optimized for interop" would have included it.